test/built-ins/Array/prototype/includes/fromIndex-equal-or-greater-length-returns-false.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.includes
+description: Return false if fromIndex >= ArrayLength
+info: |
+  22.1.3.11 Array.prototype.includes ( searchElement [ , fromIndex ] )
+
+  ...
+  4. Let n be ? ToInteger(fromIndex). (If fromIndex is undefined, this step
+  produces the value 0.)
+  5. If n ≥ 0, then
+    a. Let k be n.
+  ...
+  7. Repeat, while k < len
+    ...
+  8. Return false.
+---*/
+
+var sample = new Array(42).map(function() { return 7; });
```

Comment:
This complexity doesn't really add much value, and it's a little inconsistent with the way you seed arrays in other tests. Would you be satisfied with something like `var sample = [7, 7, 7, 7];`?